### DISCUSSION TITLE
Change Images

### DISCUSSION THREAD

**Discussion by hangouts-cmd** [community/community#62211] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Why do you have to solve a puzzle to report something?

![Screenshot 2023-07-30 7 37 14 AM](https://github.com/community/community/assets/112140400/128f6a3b-a352-4610-807a-b56ad9904084)

Also, are these images "AI generated"? I'm just curious.

**Comment by priyankarpal** [comment]
@hangouts-cmd  you're a human to prove

**Comment by GhostOf0days** [comment]
I'll propose a question to answer this.

Do you believe AI and bots should be able to mass report someone, effectively banning them as there are many reports against them, making Github's software systems believe the person is in the wrong?
